stdlib/threading.pyi

```diff
@@ -8,7 +8,84 @@ _TF = Callable[[FrameType, str, Any], Optional[Callable[..., Any]]]
 _PF = Callable[[FrameType, str, Any], None]
 _T = TypeVar("_T")
 
-__all__: list[str]
+if sys.version_info >= (3, 10):
+    __all__ = [
+        "get_ident",
+        "active_count",
+        "Condition",
+        "current_thread",
+        "enumerate",
+        "main_thread",
+        "TIMEOUT_MAX",
+        "Event",
+        "Lock",
+        "RLock",
+        "Semaphore",
+        "BoundedSemaphore",
+        "Thread",
+        "Barrier",
+        "BrokenBarrierError",
+        "Timer",
+        "ThreadError",
+        "setprofile",
+        "settrace",
+        "local",
+        "stack_size",
+        "excepthook",
+        "ExceptHookArgs",
+        "gettrace",
```

Comment:
I don't see these last two in the code on 3.9.

I do see `get_native_id`, though it looks like it's absent on some OSes: https://docs.python.org/3.10/library/threading.html#threading.get_native_id

Reply:
Oops, this is a 3.10 block and gettrace and getprofile were only added in 3.10. Sorry for that.

Reply:
Good catch on `get_native_id`. I was copy-pasting `__all__` from the source code, and didn't spot that it was conditionally appended to `__all__` lower down.

Reply:
I have added `get_native_id` to `threading.__all__`, and I have deleted `get_native_id` from `_dummy_threading.pyi`, as stubtest correctly points out that it doesn't exist in `dummy_threading` at runtime.